### PR TITLE
Update collection-index.yml to add Python/MSSQL

### PR DESCRIPTION
**Pull request by msetbar**

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -523,3 +523,8 @@
   contact: https://github.com/ElanHasson/devcontainer-features/issues
   repository: https://github.com/ElanHasson/devcontainer-features
   ociReference: ghcr.io/elanhasson/devcontainer-features
+- name: Python+MSSQL+Azure SQL Templates
+  maintainer: Mahdi Setayesh
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]: Sharing my thought 🗯️  -- As this is hosted on https://github.com/Azure-Samples org, should we rename the maintainer to something similar to 👇 ?

https://github.com/devcontainers/devcontainers.github.io/blob/847d8c04621c4639b30a3f753c212926d0f1cc7d/_data/collection-index.yml#L487 

I wonder if updating the maintainer's name from a single person to an Azure team might make it look more official and easily accepted by the community. Let me know what you think!

File: _data/collection-index.yml [hunk 2]
```diff
@@ -523,3 +523,8 @@
...
+  contact: https://github.com/Azure-Samples/python-ms-sql-devcontainer/issues
+  repository: https://github.com/Azure-Samples/python-ms-sql-devcontainer
+  ociReference: ghcr.io/Azure-Samples/python-ms-sql-devcontainer
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/azure-samples/python-ms-sql-devcontainer
```

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

**Review by samruddhikhandale — COMMENTED** [review]
Thanks for contributing, left some minor comments!

File: _data/collection-index.yml (hunk 1 above)
Comment (msetbar) [reply _data/collection-index.yml]: Done.

**Review by samruddhikhandale — APPROVED** [review]
Nice, thanks! ⚡

**Pull request merged**